mmdet/datasets/transforms/wrappers.py

```diff
@@ -170,3 +170,113 @@ def __repr__(self):
             format_string += f'{t.__class__.__name__}, '
         format_string += ')'
         return format_string
+
+
+@TRANSFORMS.register_module()
+class ProposalBroadcaster(BaseTransform):
```

Comment:
Need to explain the difference with TransformBroadcaster

Reply:
The TransformBroadcaster cannot update keys that have been updated in the TransformBroadcaster.transforms. 
If it wants to have the same effect as ProposalBroadcaster, it needs to set a lot of mapping keys, which is not easy to expansion.  ProposalBroadcaster is a more concise usage that can reduce the writing of config files